docs/rfc-1-staking-contract.adoc

```diff
@@ -544,14 +398,14 @@ For example, suppose the given staking provider has 10 T, 20 T worth of KEEP,
 and 30 T worth of NU all staked, and the maximum application authorization is
 40 T, then `getMinStaked` for that staking provider returns:
 
-* 0 T if KEEP stake type specified i.e. min = 40 T max - (10 T + 30 T worth of NU) = 0 T
-* 10 T if NU stake type specified i.e. min = 40 T max - (10 T + 20 T worth of KEEP) = 10 T
-* 0 T if T stake type specified i.e. min = 40 T max - (20 T worth of KEEP + 30 T worth of NU) < 0 T
+* 0 T if KEEP stake type specified i.e. min = 40 T max - (10 T) = 30 T
+* 10 T if NU stake type specified i.e. min = 40 T max - (10 T) = 30 T
+* 0 T if T stake type specified i.e. min = 40 T max = 40 T < 0 T
```

Comment:
Shouldn't it be:
```suggestion
* 30 T if KEEP stake type specified i.e. min = 40 T max - (10 T) = 30 T
* 30 T if NU stake type specified i.e. min = 40 T max - (10 T) = 30 T
* 40 T if T stake type specified i.e. min = 40 T max = 40 T
```
We should probably do the same change in the `getMinStaked`'s docstring.

Reply:
✔️